src/test/scala/com/advancedtelematic/director/http/DiffSpec.scala

```diff
@@ -17,7 +17,7 @@ import eu.timepit.refined.api.Refined
 
 import scala.concurrent.Future
 
-trait DiffSpec extends DirectorSpec
+trait DeviceUpdateSpec extends DirectorSpec
```

Comment:
Maybe we can consider renaming `RsaDiffSpec` and `EdDiffSpec`, as well as the file itself?